### PR TITLE
Add extra user information when migrating release

### PR DESCRIPTION
**Pull request by lunny**
It is similar with issues or comments to save original author and id on release when migrating from other git services.

### CODE REVIEW

**Review by lafriks — CHANGES_REQUESTED** [review]
Needs migration step

**Comment by lunny** [comment]
@lafriks done.

File: modules/migrations/github.go [hunk 1]
```diff
@@ -214,6 +214,11 @@ func (g *GithubDownloaderV3) convertGithubRelease(rel *github.RepositoryRelease)
 		name = *rel.Name
 	}
 
+	var email string
+	if rel.Author.Email != nil {
+		email = *rel.Author.Email
```
Comment (guillep2k) [inline modules/migrations/github.go:219]: Is it possible that this e-mail comes in the format `<username>@users.noreply.github.com`? Would that be a problem?

Comment (lunny) [reply modules/migrations/github.go:219]: @guillep2k If the author public his email, this email will be not nil, so we could copy it. If he hides his email on github, we will receive nil here. So I don't think this is a problem.

**Review by guillep2k — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**